students/Osiddiquee/lesson02/print_grid2.py

```diff
@@ -0,0 +1,16 @@
+def print_grid2(x,y):
+    border = '+'
+    interior = '|'
+    for n in range (0,x):
+        for n in range (0,y):
+            border += ' -'
+        border += ' +'
+    for n in range (0,x):
+        for n in range (0,y*2):
+            interior += ' '
```

Comment:
Instead of  range(0, y * 2) and increasing the number of loops, you can make interior += two spaces instead of one. 
Loops can be expensive computationally. Instead of loops you can do something like: 
```
border = "+" + (" -" * y + " " + "+") * x
```